docs/examples/commands.md

```diff
@@ -0,0 +1,52 @@
+# Slash Commands
+
+In this example, you'll get to know how to create commands and listen to incoming interactions.
+
+## Creating a Command
+
+First off, we need to create a command so that users can use it. We will create an `echo` command which simply returns what the user inputted. Note that global commands can take up to an hour to appear in the client, so if you want to test a new command, you should create it for one guild first.
+
+```js
+// The data for our command
+const commandData = {
+  name: 'echo',
+  description: 'Replies with your input!',
+  options: [{
+    name: 'input',
+    type: 'STRING',
+    description: 'The input which should be echoed back',
+    required: true,
+  }],
+};
+
+client.once('ready', () => {
+  // Creating a global command
+  client.application.commands.create(commandData);
+
+  // Creating a guild-specific command
+  client.guilds.cache.get('id').commands.create(commandData);
```

Comment:
Would it be possible to also impl this on the manager?

Reply:
What exactly do you mean? `Guild#commands` is a manager as well